src/main/drivers/serial_uart_pinconfig.c

```diff
@@ -53,14 +53,31 @@ void uartPinConfigure(const serialPinConfig_t *pSerialPinConfig)
         const uartHardware_t *hardware = &uartHardware[hindex];
         const UARTDevice_e device = hardware->device;
 
+#if !(defined(STM32F1) || defined(STM32F4)) // Older CPUs don't support pin swap.
+        uartdev->pinSwap = false;
+#endif
         for (int pindex = 0 ; pindex < UARTHARDWARE_MAX_PINS ; pindex++) {
-            if (hardware->rxPins[pindex].pin == pSerialPinConfig->ioTagRx[device]) {
+            if (pSerialPinConfig->ioTagRx[device] == hardware->rxPins[pindex].pin) {
```

Comment:
What is the purpose of this change?

Reply:
So that the line just below the `if` expression has the same order as the expression, i.e. `hardware->rxPins` is on the right on both lines making it easier to correlate and see the differences of the 4 expressions and the code blocks that follow them.